modules/ROOT/pages/whats-new-in-studio.adoc

```diff
@@ -1 +1,44 @@
 = What’s New in Studio
+
+Anypoint Studio 7.10 introduces a notification mechanism to help you keep up to date with your modules versions in Exchange, and also enables you to propagate metadata to flows and sub-flows.
+
+== Modules Updates Notifications
+
+Every time Studio starts up it checks for updated modules in your open projects. You can choose to update to the latest minor version of your module or upgrade directly to the latest version available.
```

Comment:
"start up" is what is called a _colocation_.  There is an entry for these in the style ref., but they are essentially two-word verbs created by including a preposition. When translated literally, they can cause problems, and they ofter don't make literal sense: why "up," for example. (I had students in Japan who were always asking me about these: Why do you "put on" but "take off" clothing? English is weird....)

So...can we replace "start up" with "initiates" or something similar?

Reply:
```suggestion
Every time Studio starts, it checks for updates to modules in your open projects. Based on what it finds, you can choose to update to the latest minor version of your module or upgrade directly to the latest version available.
```

Reply:
Great feedback. Thank you.
I'll review my other usages of colocations across my content. 👍